webapp/tests/karma/ts/services/db-sync.service.spec.ts

```diff
@@ -898,18 +898,6 @@ describe('DBSync service', () => {
   });
 
   describe('on change', () => {
-    // todo!!!
```

Comment:
This functionality was moved to `replicate` service, but I forgot to remove this test.